### PR TITLE
Add `unbase64` and `base64` Spark function

### PR DESCRIPTION
**Pull request by sujithjay**
The Spark function `unbase64` is the same as the Presto function `from_base64`. Similarly, the Spark function `base64` is the same as `to_base64` in Presto. This PR registers the Presto functions in SparkSQL.

1. https://spark.apache.org/docs/latest/api/sql/index.html#unbase64
2. https://spark.apache.org/docs/latest/api/sql/index.html#base64 
3. https://prestodb.github.io/docs/current/functions/binary.html

### CODE REVIEW

**Comment by sujithjay** [comment]
@PHILO-HE Could you please review? This PR is from our discussion in https://github.com/apache/incubator-gluten/pull/5242

File: velox/docs/functions/spark/binary.rst [hunk 1]
```diff
@@ -55,3 +55,7 @@ Binary Functions
     have a value of 224, 256, 384, 512, or 0 (which is equivalent to 256). If asking
     for an unsupported bitLength, the return value is NULL.
     Note: x can only be varbinary type.
+
+.. spark:function:: unbase64(x) -> varbinary
```
Comment (PHILO-HE) [inline velox/docs/functions/spark/binary.rst:59]: Better to document it in `math.rst`.

File: velox/functions/sparksql/Register.cpp [hunk 2]
```diff
@@ -53,6 +54,11 @@ extern void registerElementAtFunction(
     const std::string& name,
     bool enableCaching);
 
+inline void registerFromBase64Function(const std::string& prefix) {
+  registerFunction<FromBase64Function, Varbinary, Varchar >(
```
Comment (PHILO-HE) [inline velox/functions/sparksql/Register.cpp:58]: Looks no need to define `registerFromBase64Function`.

I note in Spark unbase64 & hex/unhex are in the same category. So better to register it here:
https://github.com/facebookincubator/velox/blob/93b599b94c644874957a7371c0f365345851d831/velox/functions/sparksql/RegisterArithmetic.cpp#L58

**Review by PHILO-HE — COMMENTED** [review]
Just some trivial comments. Thanks!

**Comment by konjac** [comment]
It seems `base64` is not implemented and also similar. Do you consider adding them together?

**Comment by sujithjay** [comment]
@PHILO-HE @konjac I have addressed the review comments. Please review.

**Comment by pedroerp** [comment]
(please ignore my tagging/untagging - just testing something unrelated)

**Comment by sujithjay** [comment]
Hello @PHILO-HE @konjac @Yohahaha, could you please review the changes? Thank you.

**Comment by PHILO-HE** [comment]
Hi @sujithjay, I note for unbase64, possible differences between velox & spark are listed here: https://github.com/apache/incubator-gluten/pull/5242#discussion_r1559509415. Do you have any comment on that?

**Pull request closed without merging**